### PR TITLE
Bug 1226363 - Rebase our L forks to 5.1.1_r29 r=gerard-majax

### PR DESCRIPTION
**Pull request by AdFad666**
Signed-off-by: Adam Farden <adam@farden.cz>

### CODE REVIEW

File: sony-aosp-l.xml [hunk 1]
```diff
@@ -15,10 +15,11 @@
   <project name="platform/hardware/qcom/media" path="hardware/qcom/media"/>
   <project name="platform/hardware/qcom/wlan" path="hardware/qcom/wlan"/>
   <project name="platform/hardware/ril" path="hardware/ril"/>
-  <project name="hardware_qcom_display" path="hardware/qcom/display" remote="b2g" revision="b2g-5.1.0_r1"/>
-  <project name="platform_hardware_broadcom_libbt" path="hardware/broadcom/libbt" remote="b2g" revision="b2g-5.1.1_r3"/>
+  <project name="platform_external_libnfc-nci" path="external/libnfc-nci" remote="b2g" revision="5.1.1_r29"/>
```
Comment (lissyx) [inline sony-aosp-l.xml]: b2g-5.1.1_r29 and not just 5.1.1_r29

File: sony-aosp-l.xml [hunk 2]
```diff
@@ -15,10 +15,11 @@
...
+  <project name="hardware_qcom_display" path="hardware/qcom/display" remote="b2g" revision="b2g-5.1.1_r29"/>
+  <project name="platform_hardware_broadcom_libbt" path="hardware/broadcom/libbt" remote="b2g" revision="b2g-5.1.1_r29"/>
   <remove-project name="platform/hardware/libhardware"/>
-  <project name="platform_hardware_libhardware" path="hardware/libhardware" remote="b2g" revision="sony-aosp-l"/>
+  <project name="platform_hardware_libhardware" path="hardware/libhardware" remote="b2g" revision="5.1.1_r29"/>
```
Comment (lissyx) [inline sony-aosp-l.xml]: b2g-5.1.1_r29 and not just 5.1.1_r29

Comment (AdFad666) [inline sony-aosp-l.xml]: derp derp derp
image

File: sony-aosp-l.xml [hunk 3]
```diff
@@ -15,10 +15,11 @@
...
   <project name="platform/hardware/qcom/wlan" path="hardware/qcom/wlan"/>
   <project name="platform/hardware/ril" path="hardware/ril"/>
-  <project name="hardware_qcom_display" path="hardware/qcom/display" remote="b2g" revision="b2g-5.1.0_r1"/>
-  <project name="platform_hardware_broadcom_libbt" path="hardware/broadcom/libbt" remote="b2g" revision="b2g-5.1.1_r3"/>
+  <project name="platform_external_libnfc-nci" path="external/libnfc-nci" remote="b2g" revision="b2g-5.1.1_r29"/>
+  <project name="hardware_qcom_display" path="hardware/qcom/display" remote="b2g" revision="b2g-5.1.1_r29"/>
+  <project name="platform_hardware_broadcom_libbt" path="hardware/broadcom/libbt" remote="b2g" revision="b2g-5.1.1_r29"/>
   <remove-project name="platform/hardware/libhardware"/>
-  <project name="platform_hardware_libhardware" path="hardware/libhardware" remote="b2g" revision="sony-aosp-l"/>
+  <project name="platform_hardware_libhardware" path="hardware/libhardware" remote="b2g" revision="b2g-5.1.1_r29"/>
```
Comment (lissyx) [inline sony-aosp-l.xml]: Hm maybe we should rename that branch? It's changing from sony-aosp-l to b2g-5.1.1_r29

File: sony-aosp-l.xml [hunk 4]
```diff
@@ -15,10 +15,11 @@
   <project name="platform/hardware/qcom/media" path="hardware/qcom/media"/>
   <project name="platform/hardware/qcom/wlan" path="hardware/qcom/wlan"/>
   <project name="platform/hardware/ril" path="hardware/ril"/>
-  <project name="hardware_qcom_display" path="hardware/qcom/display" remote="b2g" revision="b2g-5.1.0_r1"/>
-  <project name="platform_hardware_broadcom_libbt" path="hardware/broadcom/libbt" remote="b2g" revision="b2g-5.1.1_r3"/>
+  <project name="platform_external_libnfc-nci" path="external/libnfc-nci" remote="b2g" revision="b2g-5.1.1_r29"/>
```
Comment (lissyx) [inline sony-aosp-l.xml]: We should also rename that one?

**Pull request merged**